operatorhub/kubernetes/release-artifacts/bundle/manifests/tekton-operator-info_rbac.authorization.k8s.io_v1_role.yaml

```diff
@@ -14,3 +14,4 @@ rules:
   - configmaps
   verbs:
   - get
+  - describe
```

Comment:
Any reason to add the `describe` verb ?

Reply:
it was there since a while, lets remove it